Makefile

```diff
@@ -95,11 +95,17 @@ deps:
 	$(GO) mod vendor
 	$(GO) mod verify
 
-# Build the docker image
+# Build the docker image with official golang image
 .PHONY: docker-build
 docker-build:
+	rm -rf bin
```

Comment:
Kind of a weird side effect, can't think of a great alternative off hand though... hrm!